daemon/container_unix.go

```diff
@@ -896,8 +900,6 @@ func (container *Container) configureNetwork(networkName, service, networkDriver
 		}
 	}
 
-	container.updateSandboxNetworkSettings(sb)
```

Comment:
Thanks @mavenugo.  I've updated the PR, but until Jenkins problems are resolved we won't have validation via CI.  I have run it locally to verify it is working for basic cases.